package.json

```diff
@@ -285,46 +285,40 @@
         "title": "Language Server",
         "order": 1,
         "properties": {
-          "terraform.languageServer": {
+          "terraform.languageServer.enable": {
+            "scope": "resource",
             "order": "0",
-            "type": "object",
-            "description": "Language Server settings",
-            "properties": {
-              "external": {
-                "type": "boolean",
-                "default": true,
-                "description": "Whether an external language server binary should be launched."
-              },
-              "pathToBinary": {
-                "scope": "resource",
-                "type": "string",
-                "default": "",
-                "description": "Path to language server binary (optional)"
-              },
-              "args": {
-                "scope": "resource",
-                "type": "array",
-                "default": [
-                  "serve"
-                ],
-                "description": "Arguments to pass to language server binary"
-              }
+            "type": "boolean",
+            "default": true,
+            "description": "Enable/disable Terraform Language Server"
```

Comment:
Is there a particular reason behind the "disable" in the description? I mean if it's a tickbox/boolean I'd expect it to express what the ticked/`true` value means.

<img width="307" alt="Screenshot 2022-06-23 at 12 35 53" src="https://user-images.githubusercontent.com/287584/175280384-fba19dd3-6d57-4771-9625-59e0bb5723f7.png">
<img width="324" alt="Screenshot 2022-06-23 at 12 37 08" src="https://user-images.githubusercontent.com/287584/175280342-5c23d861-ad0b-42a2-9219-dffa05c52191.png">

Reply:
I don't know what other wording to use here.

Reply:
Dropped disable